File: 백준/Bronze/11022. A＋B － 8/A＋B － 8.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        for(int i = 1; i <= n; i++){
            int a = in.nextInt();
            int b = in.nextInt();
            System.out.printf("Case #%d: %d + %d = %d\n",i, a, b, a+b);
        }
    }
}

File: 백준/Bronze/11022. A＋B － 8/A＋B － 8.kt
import java.util.Scanner
fun main(){
    val sc = Scanner(System.`in`)
    for(i in 1..(sc.nextInt())){
        var a = sc.nextInt(); var b = sc.nextInt()
        println("Case #${i}: ${a} + ${b} = ${a+b}")
    }
}